src/hermite/cubic.jl

```diff
@@ -0,0 +1,162 @@
+struct CubicHermite
```

Comment:
```suggestion
"""
     CubicHermite
     
This type is purposely left undocumented since the interface is expected to radically change in order to make it conform to the `AbstractInterpolation` interface. Consider this API to be highly unstable and non-public, and that breaking changes to this code could be made in a point release.
"""
struct CubicHermite
```